compiler/main/driver.cpp

```diff
@@ -1584,6 +1597,35 @@ static void checkUnsupportedConfigs(void) {
   }
 }
 
+static void checkRuntimeBuilt(void) {
+  // no need for a runtime to be built for chpldoc
+  if (fDocs) {
+    return;
+  }
```

Comment:
Should we also skip this if we're skipping `makeBinary` (through `--stop-after-pass` or `--no-codegen`)?

Reply:
Makes sense to me - I'll look at adding that.